python/graphstorm/inference/np_infer.py

```diff
@@ -19,14 +19,14 @@
 import logging
 
 from .graphstorm_infer import GSInferrer
-from ..model.utils import save_embeddings as save_gsgnn_embeddings
+from ..model.utils import save_shuffled_node_embeddings
 from ..model.utils import save_node_prediction_results
-from ..model.utils import shuffle_predict
+from ..model.utils import NodeIDShuffler
```

Comment:
Not sure why, but the `NodeIDShuffler` code does not show up as a new commit.

Reply:
It was introduced in https://github.com/awslabs/graphstorm/pull/481

Reply:
I saw the code, but don't know why I cannot leave comments on that code. The code of NodeIDShuffler class is not in the green background that a newly added code should be in.